install/src/main/java/org/jetbrains/bsp/bazel/install/ProjectViewCLiOptionsProvider.kt

```diff
@@ -0,0 +1,63 @@
+package org.jetbrains.bsp.bazel.install
+
+import ch.epfl.scala.bsp4j.BuildTargetIdentifier
+import io.vavr.control.Try
+import org.jetbrains.bsp.bazel.install.cli.CliOptions
+import org.jetbrains.bsp.bazel.install.cli.ProjectViewCliOptions
+import org.jetbrains.bsp.bazel.projectview.generator.DefaultProjectViewGenerator
+import org.jetbrains.bsp.bazel.projectview.model.ProjectView
+import org.jetbrains.bsp.bazel.projectview.model.sections.*
+
+class ProjectViewCLiOptionsProvider {
+    fun generateProjectViewAndSave(cliOptions: CliOptions): Try<ProjectView> {
+        val generator = DefaultProjectViewGenerator()
+        val a = generateProjectView(cliOptions)
+        return generator.generatePrettyStringAndSaveInFile(a, cliOptions.projectViewFilePath
+                ?: cliOptions.workspaceRootDir.resolve("projectview.bazelproject"))
+                .map { a }
+    }
+
+    private fun generateProjectView(cliOptions: CliOptions): ProjectView {
+        val projectViewCliOptions = cliOptions.projectViewCliOptions
+        return ProjectView(
+                javaPath = javaPathSection(projectViewCliOptions),
```

Comment:
u can just replace it with javaPath = javaPathSection(cliOptions.projectViewCliOptions)